README.md

```diff
@@ -1,66 +1,40 @@
-    * Not update, you can view the original file but the results will not be loaded
-
-### Sarif Explorer
- * Automatically launches when the first Sarif file is opened
- * Updates the Result Details Panel with the currently selected result in the Results List, Problems Panel, or in source code
- * Manually open it by typing "Sarif: Launch the Sarif Explorer" in the Command Palette(F1) or using the hotkey (Ctrl+L then Ctrl+E)
-
-#### Results List
-![Demo](/README/ResultsList.gif?raw=true)
- * Available columns: Baseline State, Message, Result File, Position, Rule Id, Rule Name, Run Id, Sarif File, Severity, Kind, Rank, Tool, Automation Category, Automation Id, *new*Logical Location
- * Group By: Results can be grouped by a column
-    * Groups are sorted by number of results in each group
- * Sort By: Results are sortable by clicking the column header
- * Filter: Show/Hide the Filter input area by clicking the Filter icon
-    * Toggle button for toggling Match Case
-    * No wildcard support yet
- * Hide/Show columns: Visibility of each column can be toggled by clicking the Eye icon
- * Clicking a result in the list will navigate to the source and display the details in the Sarif Explorer
- * Persistence: Group By, Sort By, and Hidden columns are persisted in settings
-
-### API
+[![](https://vsmarketplacebadge.apphb.com/version-short/MS-SarifVSCode.sarif-viewer.svg)](https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer)
+[![](https://vsmarketplacebadge.apphb.com/downloads-short/MS-SarifVSCode.sarif-viewer.svg)](https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer)
+
+# SARIF Viewer for Visual Studio Code
+
+A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/VSCode) that adds support for viewing [SARIF](http://sarifweb.azurewebsites.net/) logs. SARIF log results can be viewed as squiggles in your source, in the Problems list, or in a dedicated **SARIF Results Panel**. The **SARIF Results Panel** offers richer grouping, filtering, column, and details options over the standard Problems list.
+
+![overview](.github/README.hero.png)
+
+## What's New?
+
+Version 3 incorporates many feedback-based improvements:
+* Improved keyboard accessibility within the **SARIF Results Panel**. Arrow key through the list of results.
+* Resizable details section within the **SARIF Results Panel**.
+* Generally improved performance and responsiveness.
+* Automatic reconciliation of URIs between the SARIF log and your local workspace in most cases.
+
+To focus our efforts, we have dropped some less-used and less-reliable features:
+* Support for old SARIF versions - We now strictly support the current version (2.1.0 rtm.5) and version back (2.1.0 rtm.4). Older versions can be upgraded with the standalone SARIF Multitool (via [nuget](https://www.nuget.org/packages/Sarif.Multitool/) and [npm](https://www.npmjs.com/package/@microsoft/sarif-multitool)).
```

Comment:
>(2.1.0 rtm.5) and version back (2.1.0 rtm.4) [](start = 79, length = 44)

Don't even mention it. Just say "2.1.0". If they load an rtm.4 file and we don't complain, that's just gravy.

Reply:
Oh, and it's not just the "current" version. We don't want people wondering when the "next" version is coming out. Let's say "support the public standard version 2.1.0".

---
In reply to: [483872733](https://github.com/microsoft/sarif-vscode-extension/pull/349#discussion_r483872733) [](ancestors = 483872733)

Reply:
We do not support 2.1.0 rtm.0 through rtm.3. I assume you're implying it's still okay to claim "support the public standard version 2.1.0". Doesn't seem people will care.